akka-projection-core/src/main/scala/akka/projection/internal/OffsetStrategy.scala

```diff
@@ -79,7 +79,7 @@ private[projection] final case class GroupedHandlerStrategy[Envelope](
  */
 @InternalApi
 private[projection] final case class FlowHandlerStrategy[Envelope](
-    flowCtx: FlowWithContext[Envelope, Envelope, Done, Envelope, _])
+    flowCtx: FlowWithContext[Envelope, OpenEnvelope[_, Envelope], Done, OpenEnvelope[_, Envelope], _])
```

Comment:
It's somewhat unfortunate to introduce another type in the user API. Since this is the only place it's visible to the users we should maybe name it `ProjectionContext` or `EnvelopeContext`?